kotlinx-coroutines-core/common/src/flow/operators/Delay.kt

```diff
@@ -14,13 +14,30 @@ import kotlinx.coroutines.selects.*
 import kotlin.jvm.*
 import kotlin.time.*
 
+/* Scaffolding for Knit code examples
+<!--- TEST_NAME FlowDelayTest -->
```

Comment:
Same here, would be nice to hide it in an external file

Reply:
Yes. That would be a nice addition. Created https://github.com/Kotlin/kotlinx-knit/issues/18 and https://github.com/Kotlin/kotlinx-knit/issues/18